apk-generator/v2/app/utils/notification.py

```diff
@@ -46,7 +80,24 @@ def send_mail_via_smtp_(config, payload):
         mailer.start()
         message = Message(author=payload['from'], to=payload['to'])
         message.subject = payload['subject']
-        message.plain = strip_tags(payload['html'])
+        message.plain = payload['message']
         message.rich = payload['html']
```

Comment:
There is no param called `html`. Its called as `message`. Put that in rich and use strip tags function on it and use that for plain.